build-system/tasks/helpers.js

```diff
@@ -163,6 +164,12 @@ async function compileAllJs(options) {
     doBuildJs(jsBundles, 'amp-inabox.js', options),
   ]);
   await compileCoreRuntime(options);
+
+  // TODO(alanorozco): Is there a cleaner way to create this alias?
```

Comment:
I don't think `compileAllJs` should have to worry about this detail of implementation. A better place for this code is where the `latestName` handling is done.

**Unminified:** Copy `bento.max.js` to `custom-elements-polyfill.max.js` here (after line 379).
https://github.com/ampproject/amphtml/blob/59dc7d08063af00b831c719251438278329f9bb7/build-system/tasks/helpers.js#L351-L380

**Minified:** Copy `bento.js` to `custom-elements-polyfill.js` here (after line 327).
https://github.com/ampproject/amphtml/blob/59dc7d08063af00b831c719251438278329f9bb7/build-system/tasks/helpers.js#L294-L328

Reply:
`latestName` is exactly what I want! I'm using that now.

Given the existing semantics of the option, do you think it's worth it to rename to `aliasName`? I can do it on a separate PR if that's better.

Reply:
Agree. Assuming the community no longer refers to components using the `latest` moniker, I'm good with replacing the option with `aliasName`. (Is it worth doing a traffic study to see how many pages actually use a `-latest` extension?)

Separate PR for the option rename followed by rebasing of this PR to implement the `bento.js` change SGTM.

Thank you!